website/templates/include/comment_template.mako

```diff
@@ -124,42 +107,45 @@
                                 </div>
                             </div>
                         </div>
-
                     </div>
 
-                    <div>
-
+                    <div data-bind="ifnot: mode === 'widget'">
+                        
```

Comment:
There are a couple of places in this file with extraneous space like this. It's not worth another commit to fix, but please watch for it in the future.